quickstack/manifests/cinder.pp

```diff
@@ -27,6 +27,14 @@
   $manage_service = true,
   $debug          = false,
   $verbose        = false,
+  $auth_uri        = 'http://localhost:5000/v2.0',
+  $identity_uri    = 'http://localhost:35357/v2.0',
+  $auth_protocol   = 'http',
+  $cert_file       = '/etc/pki/tls/certs/cinder.crt',
```

Comment:
check if its possible to remove auth_protocol and determine it based on "use_ssl" flag.